<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT" verbose="0">
  <listeners>
    <listener class-name="testComponents.Listeners"/>
  </listeners>
  <test thread-count="5" parallel="tests" name="ExceptionHandling(failed)" verbose="0">
    <classes>
      <class name="tests.ErrorValidations">
        <methods>
          <include name="LoginErrorValidation"/>
          <include name="teardown"/>
          <include name="LaunchApplication"/>
        </methods>
      </class> <!-- tests.ErrorValidations -->
    </classes>
  </test> <!-- ExceptionHandling(failed) -->
  <test thread-count="5" parallel="tests" name="SubmitOrder(failed)" verbose="0">
    <classes>
      <class name="tests.StandAlone">
        <methods>
          <include name="teardown"/>
          <include name="SubmitOrder"/>
          <include name="OrderHistoryTest"/>
          <include name="LaunchApplication"/>
        </methods>
      </class> <!-- tests.StandAlone -->
    </classes>
  </test> <!-- SubmitOrder(failed) -->
</suite> <!-- Failed suite [Suite] -->
